cognite/client/_api/datapoints_subscriptions.py

```diff
@@ -263,7 +263,7 @@ def iterate_data(
                 "externalId": external_id,
                 "partitions": [p.dump(camel_case=True) for p in current_partitions],
                 "limit": limit,
-                "timeoutSeconds": 5, # keep the request open until data is available, up to 5 seconds
+                "timeoutSeconds": 5,  # keep the request open until data is available
```

Comment:
This seems super useful! Accept it as an argument to the function with default being 5? 🚀